app/validation/validators.py

```diff
@@ -14,7 +14,7 @@
 logger = get_logger()
 
 
-class NumberCheck(object):
+class NumberCheck():
```

Comment:
Now this _really_ is a nit pick.

You don't need the `()` to define a class that inherits directly from `object`.

```
class NumberCheck:
   ...
```